### PR TITLE
QA - Readme: edit tests sections

### PR DESCRIPTION
**Pull request by RobotRogue**
- Edited the Tests section to clarify it more for Partners or people new to the scaffold and its use.
- Did a few style tweaks. 
- Added some text detail to explain things better.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,25 +7,47 @@ manage node and npm versions.
...
 npm run add:component
 ```
 
+
+## Pushing Bundles to Cloud (Creating a Bundle)
+
+From the project folder, run the following command to push a bundle up to the Mobify Cloud:
+
+```
+npm run push -- --m "Mobile x.x.x:GitSHA,BRANCH"
```
Comment (ellenmobify) [inline README.md]: Not sure about the inclusion of `Mobile` here. Does it need to say Mobile? Could it be something else?

File: README.md [hunk 2]
```diff
@@ -6,25 +7,47 @@ manage node and npm versions.
...
+```
+
+Including the Git Commit SHA and the branch you are pushing to cloud in the bundle message is recommended.
```
Comment (ellenmobify) [inline README.md]: Make this imperative.

`Include the Git Commit SHA and the branch you are pushing in the bundle message.`

File: README.md (hunk 1 above)
Comment (RobotRogue) [inline README.md]: After our discussion, I removed that text. It was legacy information.

File: README.md (hunk 2 above)
Comment (RobotRogue) [inline README.md]: Modified the section to make the entire message required, and explained its formatting.

File: README.md [hunk 3]
```diff
@@ -6,25 +7,49 @@ manage node and npm versions.
...
+
+## Pushing Bundles to Cloud (Creating a Bundle)
+
+From the project folder, run the following command to push a bundle up to the Mobify Cloud:
+
+```
+npm run push -- --m "VersionNum:GitSHA,BRANCH"
+```
+
+The Bundle Message is required. The format of the message is as follows:
```
Comment (ellenmobify) [inline README.md]: yassss

File: README.md [hunk 4]
```diff
@@ -1,43 +1,79 @@
-# Platform Scaffold
```
Comment (johnboxall) [inline README.md]: @RobotRogue did we mean to delete this section?

**Comment by johnboxall** [comment]
@RobotRogue @ellenmobify – we need to pick a strategy between the root README and the one in the `web` directory – we're duplicating things between them right now and it isn't clear why things are in one or the other.

**Pull request merged**